test cases/common/234 very long commmand line/seq.py

```diff
@@ -1,6 +1,19 @@
 #!/usr/bin/env python3
+"""
```

Comment:
Perhapy it would make sense to rename this script, since it's no longer doing the same thing as coreutils `seq`.

Reply:
I've made it name_gen.py